web/package.json

```diff
@@ -9,7 +9,6 @@
     "build": "npm run --prefix scripts copy-keys && ng build -c $npm_config_config",
     "watch": "npm run build -- --watch",
     "start": "npm run build --config=$npm_config_config && ng serve -c $npm_config_config",
-    "pretest": "./pretest.sh",
```

Comment:
@rfontanarosa Sorry I misunderstand which `pre-` this was today when we spoke. This is still needed.